File: jour_6/jour6.pas
program ProgramName;
var 
    number : integer;
begin
    write('Entrez un nombre entier : ');
    readln(number);
    if (number mod 2 = 0) then
        begin
            write('le nombre est pair');
        end
    else 
        begin
            write('le nombre est impair ');
        end;
end.

File: jour_6/jour6.c
#include<stdio.h>
int number;
int main(){
    printf("Entrez un nombre entier : ");
    scanf("%d",&number);
    if (number % 2 == 0){
        printf("le nombre est pair\n");
    }
    else {
        printf("le nombre est impair\n");
    }
    return 0;
}    